pkg/controllers/nodeclass/controller.go

```diff
@@ -93,28 +93,28 @@ func (c *Controller) Reconcile(ctx context.Context, nodeClass *v1beta1.EC2NodeCl
 		v1beta1.AnnotationEC2NodeClassHashVersion: v1beta1.EC2NodeClassHashVersion,
 	})
 
-	err := multierr.Combine(
+	errs := multierr.Combine(
 		c.resolveSubnets(ctx, nodeClass),
 		c.resolveSecurityGroups(ctx, nodeClass),
 		c.resolveAMIs(ctx, nodeClass),
 		c.resolveInstanceProfile(ctx, nodeClass),
 	)
 	if lo.FromPtr(nodeClass.Spec.AMIFamily) == v1beta1.AMIFamilyAL2023 {
-		if cidrErr := c.launchTemplateProvider.ResolveClusterCIDR(ctx); err != nil {
-			err = multierr.Append(err, fmt.Errorf("resolving cluster CIDR, %w", cidrErr))
+		if cidrErr := c.launchTemplateProvider.ResolveClusterCIDR(ctx); cidrErr != nil {
```

Comment:
nbd, but if we're calling the mutli-err, `errs`, can we still just leave this as `err`?